uiowa/uiowa
# People list block link fix

Resolves https://github.com/uiowa/uiowa/issues/5666. 

# How to test

- `ddev blt ds --site=sandbox.uiowa.edu`
- Go to https://sandbox.prod.drupal.uiowa.edu/people-examples and verify that images are clickable. 
- Edit the list block and make sure that the "Hide fields" option does not contain a "Content: link to content" option. 
- Verify that is OK to remove https://github.com/uiowa/uiowa/pull/5710/commits/5d0de989c4f1c47b4eee7526df4f54e613c0d3c3.
